tests/test_configuration.py

```diff
@@ -490,6 +490,20 @@ def test_is_nested():
         assert not eq.is_nested(grid=grid, msg=None)
 
 
+@pytest.mark.unit
+def test_is_nested_theta():
+    """Test that new version of is nested also catches messed up theta contours."""
+    eq = Equilibrium(L=6, M=6, N=0, iota=1)
+    # just mess with lambda, so rho contours are the same
+    eq.L_lmn += 1e-1 * np.random.default_rng(seed=3).random(eq.L_lmn.shape)
+    grid = QuadratureGrid(10, 10, 0, NFP=eq.NFP)
+    g1 = eq.compute("sqrt(g)", grid=grid)["sqrt(g)"]
+    g2 = eq.compute("sqrt(g)_PEST", grid=grid)["sqrt(g)_PEST"]
+    assert np.all(g1 > 0)  # regular jacobian will still be fine
+    assert np.any(g2 < 0)  # PEST jacobian should be negative
+    assert not eq.is_nested()
+
+
```

Comment:
Did you want to add that plasma volume test?

```python
data = eq.compute(["V", "V(r)"], grid)
V_PEST = np.sum(data["sqrt(g)_PEST"] * (1 + data["lambda_t"]) * transforms["grid"].weights)
np.testing.assert_allclose(data["V"], V_PEST)
# or maybe
# divergence theorem: integral(dV div [0, 0, Z]) = integral(dS dot [0, 0, Z])
V_of_r_PEST = np.abs(surface_integrals(grid,
    (1 + data["lambda_t"]) * cross(data["e_theta_PEST"], data["e_zeta"])[:, 2] * data["Z"],
))
np.testing.assert_allclose(data["V(r)"], V_of_r_PEST)
```

Reply:
The strange Jacobian is a consequence of non-monotonicity between theta and theta_PEST. So maybe we could add a function that checks that instead of varying lambda to make the Jacobian fail. Not urgent, though.

Reply:
I didn't add the volume test because when you add in the `(1+ lambda_t)` term it cancels the 1/(1+ lambda_t) term in the PEST jacobian and the overall thing becomes exactly the same as our regular jacobian, so it's not really testing anything.